torch/nn/_functions/thnn/auto.py

```diff
@@ -276,6 +276,7 @@ def _generate_function_classes(scope_dict):
         'GRUFused',
         'LSTMFused',
         'unfolded',
+        'Im2Col',
```

Comment:
Is there any reason why this binding can't be auto-generated?

Reply:
I would think this should be auto-generated, but if I delete this line I get an import error `RuntimeError: class loader couldn't access torch.nn._functions.thnn module`.  Is there another update I am missing?